qupath-core-processing/src/main/java/qupath/lib/plugins/objects/TileClassificationsToAnnotationsPlugin.java

```diff
@@ -97,7 +97,7 @@ public Collection<Class<? extends PathObject>> getSupportedParentObjectClasses()
 	@Override
 	protected void addRunnableTasks(final ImageData<T> imageData, final PathObject parentObject, List<Runnable> tasks) {
 		ParameterList params = getParameterList(imageData);
-		if (params.getBooleanParameterValue("clearAnnotations")) {
+		if (params.getBooleanParameterValue("clearAnnotations") || params.getBooleanParameterValue("removeAnnotations")) {
```

Comment:
Was hoping this would be sufficient to avoid things breaking between versions, but I think I'm wrong